frontend/src/shared/components/SmileNavBar.tsx

```diff
@@ -19,9 +24,26 @@ const SmileNavBar: FunctionComponent = () => {
           <NavLink href="/patients">Patients</NavLink>
           <NavLink href="/samples">Samples</NavLink>
         </Nav>
+        {userEmail && (
+          <div className="ms-auto d-flex">
+            <p className="m-auto">Logged in as {userEmail}</p>
+            <button
+              type="button"
+              className="btn btn-outline-primary btn-sm m-3"
+              onClick={() => {
+                fetch("https://localhost:4000/logout", {
```

Comment:
avoid embedding network request inside JSX callback.  You should instead declare a function and call it.